saleor/core/management/commands/populatedb.py

```
@@ -88,54 +100,62 @@ def handle(self, *args, **options):
             "saleor.payment.gateways.dummy_credit_card.plugin."
             "DummyCreditCardGatewayPlugin",
         ]
-        create_images = not options["withoutimages"]
+        create_images = not options["withoutimages"] 
+        print_messages = options["verbose"] 
+        multiplier = options["multiplier"]
+
+        def print_message(msg): 
+            if print_messages: 
+                self.stdout.write(msg) 
+        print('Multiplier:', multiplier)
```

Ruff

`print` found

Remove `print`

(T201)